### PR TITLE
[Feat]: Authetication => login & signup page added

### PR DESCRIPTION
**Pull request by t007rushi**
Login Page Added
 ->  login & signup button
Signup Page added
-> user then login feat
form & side image to showcase advt.

### CODE REVIEW

File: components/autheticate.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+/* form image */
+.form-img {
+  height: 100%;
+  width: 40%;
+  border-radius: 1rem 0 0 1rem;
+}
+
+.form-block {
+  gap: 0.5rem;
```
Comment (sks-21) [inline components/autheticate.css:23]: for values less than 10px, rems are not preferred

File: components/autheticate.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.form-input-txt {
+  width: 20rem;
+}
+
+.form-input-box {
+  padding: 0.7rem 0;
```
Comment (sks-21) [inline components/autheticate.css:45]: same as above

File: screens/LoginPage.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://pushpa-ui.netlify.app/documentation/doc.css"
+    />
+    <link rel="stylesheet" href="../components/main.css" />
+    <link rel="stylesheet" href="./autheticate.css" />
```
Comment (sks-21) [inline screens/LoginPage.html:17]: typo in 'authenticate'. You may change the file name

**Review by sks-21 — APPROVED** [review]
UI looks really cool.
Suggested some changes. Rest all is good to merge.

File: screens/LoginPage.html (hunk 3 above)
Comment (t007rushi) [reply screens/LoginPage.html:17]: This thing a dev can't run from😥

**Review by horsemaker — APPROVED** [review]

**Review by ozairkhan2810 — APPROVED** [review]
everything looks good, can merge

**Pull request merged**